google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -173,6 +178,40 @@ public LogMetric apply(MetricInfo metricInfo) {
   private LoggingRpc loggingRpcMock;
   private Logging logging;
 
+  private void configureListLogsTests(List<String> returnedList, String cursor) {
```

Comment:
please document both overloads of configureListLogsTests so the reader doesn't need to scroll into the implementation and figure out the differences by reading code in both methods.

Reply:
Will be done